### DISCUSSION TITLE
High memory usage from from 37.387.2

### DISCUSSION THREAD

**Discussion by m15o** [renovatebot/renovate#29448] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

circleci.com

### Please tell us more about your question or problem

We are using renovate container `renovate/renovate:latest.`
From version `37.387.2`, renovate is killed by os because of out of memory.

It probably because re2 issue.
https://github.com/uhop/node-re2/issues/210 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Can you confirm if it's that specific patch which made it worse? Or a range of releases?

**Reply by m15o** [reply]
I can confirm that until version `37.387.0` or below, memory usage is normal. (average 2gb memory usage in 4gb system)
And version `37.387.2` or avobe, I got out of memory error (even at at 6gb system.)

I dont have version `37.387.1` result. I suspect `37.387.2` because of the re2 issue.

**Reply by m15o** [reply]
https://github.com/renovatebot/renovate/pull/29455 fixed the problem!
Thank you
